stateproof/builder.go

```diff
@@ -155,6 +177,13 @@ func (spw *Worker) createBuilder(rnd basics.Round) (builder, error) {
 		return builder{}, err
 	}
 
+	err = spw.db.Atomic(func(_ context.Context, tx *sql.Tx) error {
```

Comment:
Should we still continue on error, considering that we now need the builder for signing?